### ISSUE TITLE
Trying to Sshuttle to Mercuryo

### ISSUE THREAD

**Issue by dmitri-alleo** [threefoldtech/test_feedback#254]
As the VMs IP was whitelisted in order to connect to the Mercuryo sandbox for testing the TFTs purchase on the Testnet,

after generating the wallet and trying to sshuttle to Mercuryo
```
sshuttle -r root@185.206.122.151 0.0.0.0/0 --dns -v
```

getting the password request
```
root@AlleoShuttle1:~/js-ng# sshuttle -r root@185.206.122.151 0.0.0.0/0 --dns -v -vvvv
Starting sshuttle proxy.
firewall manager: Starting firewall with Python version 3.8.10
firewall manager: ready method name nat.
IPv6 enabled: False
UDP enabled: False
DNS enabled: True
User enabled: False
Binding redirector: 12300
TCP redirector listening on ('127.0.0.1', 12300).
TCP redirector listening with <socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 12300)>.
Binding DNS: 12300 12299
DNS listening on ('127.0.0.1', 12299).
DNS listening with <socket.socket fd=7, family=AddressFamily.AF_INET, type=SocketKind.SOCK_DGRAM, proto=0, laddr=('127.0.0.1', 12299)>.
Starting client with Python version 3.8.10
c : connecting to server...
c : executing: ['ssh', 'root@185.206.122.151', '--', 'exec /bin/sh -c \'P=python3; $P -V 2>/dev/null || P=python; exec "$P" -c \'"\'"\'import sys, os; verbosity=5; sys.stdin = os.fdopen(0, "rb"); exec(compile(sys.stdin.read(1317), "assembler.py", "exec"))\'"\'"\'\'']
c :  > channel=0 cmd=PING len=7 (fullness=0)
root@185.206.122.151's password: 
```

**Comment by xmonader** [comment]
quoting @dmitri-alleo 
> @xmonader thanks, made me realise I'm doing it the wrong way around, I was trying to ssh from the sshuttle machine instead going through it from the local, 
I guess it's working as expected, will close the issue

**Comment by dmitri-alleo** [comment]
When trying to use sshuttle to connect via the whitelisted host, getting this error

```
sw-alleo:~ $ sudo sshuttle -r root@188.206.122.151 0.0.0.0/0 --dns -v -vvvv
[sudo] password for sw:           
Starting sshuttle proxy.
firewall manager: Starting firewall with Python version 3.8.10
firewall manager: ready method name nat.
IPv6 enabled: False
UDP enabled: False
DNS enabled: True
User enabled: False
Binding redirector: 12300
TCP redirector listening on ('127.0.0.1', 12300).
TCP redirector listening with <socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 12300)>.
Binding DNS: 12300 12299
DNS listening on ('127.0.0.1', 12299).
DNS listening with <socket.socket fd=7, family=AddressFamily.AF_INET, type=SocketKind.SOCK_DGRAM, proto=0, laddr=('127.0.0.1', 12299)>.
Starting client with Python version 3.8.10
c : connecting to server...
c : executing: ['ssh', 'root@188.206.122.151', '--', 'exec /bin/sh -c \'P=python3; $P -V 2>/dev/null || P=python; exec "$P" -c \'"\'"\'import sys, os; verbosity=5; sys.stdin = os.fdopen(0, "rb"); exec(compile(sys.stdin.read(1317), "assembler.py", "exec"))\'"\'"\'\'']
c :  > channel=0 cmd=PING len=7 (fullness=0)
```

**Comment by xmonader** [comment]
Should this issue be closed, given that you were actually able to #255  ?

**Comment by dmitri-alleo** [comment]
Yes, thank you @xmonader
